auditing/auditing-interceptor.go

```diff
@@ -323,9 +325,10 @@ func (i auditingConnectInterceptor) WrapUnary(next connect.UnaryFunc) connect.Un
 	}
 }
 
-func NewConnectInterceptor(a Auditing, logger *zap.SugaredLogger, shouldAudit func(fullMethod string) bool) connect.Interceptor {
+func NewConnectInterceptor(a Auditing, logger *slog.Logger, shouldAudit func(fullMethod string) bool) connect.Interceptor {
 	if a == nil {
-		logger.Fatal("cannot use nil auditing to create connect interceptor")
+		logger.Error("cannot use nil auditing to create connect interceptor")
+		// FIXME return something useful
```

Comment:
Shouldn't you do at least a `panic` or exit with non-zero value?

Reply:
done